### DISCUSSION TITLE
Clarification on "Output table description"

### DISCUSSION THREAD

**Discussion by nicolechaumont** [GoekeLab/xpore#135] [Q&A] [answered]
Hello, 

Could you give me few details on the following issues 

1. diff_mod_rate_<condition1>_vs_<condition2> [from the definition](https://github.com/GoekeLab/xpore/blame/master/docs/source/outputtable.rst#L12) it looks that this value could be log2 ratio of condition1/ condition2. Is it the case ? 

2. The last column of the table in your doc is ["mod_assignment"](https://github.com/GoekeLab/xpore/blame/master/docs/source/outputtable.rst#L22). However, using diffmod  0.5.6 we have supplementary column "t-test". What is it corresponding to ? 

3. If we work on output table using the p-value in column 6 "pval_<condition1>_vs_<condition2>" to extract significant values do we have to correct the values for multitesting (BH or p-adjust )

Thank you for your help.

Nicole

**Comment by jonathangoeke** [comment, marked as answer]
Hi @nicolechaumont 

We highly recommend to use the latest version of xPore, please update your installation.

1) The differential modification rate is the difference in the modifiation rates between conditions
2) The t-test is used as an optional pre-filter to quickly discard sites that are unilkley to be differentially modified from the analysis. The t-test makes it faster to run xPore (at lower accuracy)
3) The p-values are not adjusted, you could apply aBH adjustment. We primarily recommend to use the p-values/z-scores for ranking and focus the analysis on the top predicted sites rather than a fixed p-value threshold.

I hope that helps!
Jonathan
